docs/get-started/getting-started-gcp.md

```diff
@@ -74,7 +74,10 @@ The script below will initialize with the following:
 ### Create a Dataproc Cluster using T4's
 * One 16-core master node and 5 32-core worker nodes
 * Two NVIDIA T4 for each worker node
-
+Note that currently for image-version 2.1, not all operating systems is available for installing GPU drivers with Secure Boot, please
```

Comment:
A few nits, but generally it looks really good.  Perhaps we could have something more like.

```
Note that with image-version 2.1 Secure Boot is enabled by default and not all operating systems currently have
support for installing GPU drivers that are properly signed.  Please follow this 
[document](https://cloud.google.com/compute/docs/gpus/install-drivers-gpu#secure-boot) for more details on how
to setup supported operating systems with secure boot and signed GPU drivers.  For unsupported operating
systems you can disable secure boot by adding `--no-shielded-secure-boot` while creating the cluster. This
should allow the cluster to boot correctly with unsigned GPU drivers.
```

Reply:
thanks, updated, could you please help review again?